fms/utils/tokenizers.py

```diff
@@ -67,6 +70,9 @@ def convert_ids_to_tokens(self, ids: torch.LongTensor):
         return [chr(i) for i in ids]
 
     def convert_tokens_to_ids(self, tokens: Union[str, list[str]]):
+        if isinstance(tokens, str):
+            # returning a single integer to be compatible with other tokenizers
+            return [ord(t) if ord(t) < 256 else 0 for t in tokens][0]
```

Comment:
since here we're assuming 'tokens' is a single character it might be clearer to just do something like:
```
token_id = ord(tokens[0])
return token_id if token_id < 256 else 0
```
for this str case, we might also want to add an assertion that len(tokens) == 1